lua/bqf/preview/border.lua

```diff
@@ -18,26 +18,14 @@ function Border:build(o)
     self.__index = self
     self.floatwin = FloatWin
     self.chars = o.chars
+    self.show_title = o.show_title
```

Comment:
1. `show_title` -> `showTitle`;
2. Should pass `showTitle` to `PreviewSession` and `border:build` .